cypress/e2e/scenario-1-environment.cy.js

```diff
@@ -166,56 +155,49 @@ describe("Environment", () => {
     cy.get('[aria-label="Name-input"]').clear();
     cy.get('[aria-label="Name-input"]').type("New Value Name");
     cy.get('[aria-label="Name-submit-edit"]').click();
-    cy.wait("@postEnvEdit");
-    cy.wait("@getEnv");
     cy.get('[aria-label="Name-value"]').should("contain", "New Value Name");
     //change Description value
-    cy.get('[aria-label="Description-toggle-edit"]').click();
+    cy.wait(1000);
+    cy.get('[aria-label="Description-toggle-edit"]:enabled').click();
```

Comment:
cy.get('[aria-label="Description-toggle-edit"]:enabled', { timeout: 20000 }).click();

Reply:
I tried even with 30s timeout it didn't work